### PR TITLE
feat(app_backup): Add walletId field to AppBackup class, access exter…

### PR DESCRIPTION
**Pull request by noahsilencelab**
- Add "walletId" field to AppBackup class
- Increase version of AppBackup to 2 and set "walletId" according the version

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -49,10 +49,10 @@ dependencies:
   share_plus: ^7.1.0
   file_picker: ^5.5.0
   dart_2_party_ecdsa:
-    # path: ../silent-shard-flutter-sdk # NOTE: For debugging purposes
-    git:
-      url: git@github.com:silence-laboratories/silent-shard-flutter-sdk.git
-      ref: staging
+    path: ../silent-shard-flutter-sdk # NOTE: For debugging purposes
```
Comment (dakshsilence) [inline pubspec.yaml]: Please revert these changes

File: lib/utils.dart [hunk 2]
```diff
@@ -90,6 +90,6 @@ class Version {
 
 extension StringExtension on String {
   String capitalize() {
-    return "${this[0].toUpperCase()}${substring(1).toLowerCase()}";
+    return length > 1 ? "${this[0].toUpperCase()}${substring(1).toLowerCase()}" : this[0].toUpperCase();
```
Comment (dakshsilence) [inline lib/utils.dart:96]: What if length == 1?

Comment (NoahJinnn) [reply lib/utils.dart:96]: So we capitalize that letter only

**Review by dakshsilence — APPROVED** [review]
